provider/resources.go

```diff
@@ -3133,6 +3137,7 @@ func Provider() tfbridge.ProviderInfo {
 
 			"azurerm_orchestrated_virtual_machine_scale_set": {Tok: azureDataSource(azureCompute, "getOrchestratedVirtualMachineScaleSet")},
 			"azurerm_container_app":                          {Tok: azureDataSource(azureContainerApp, "getApp")},
+			"azurerm_ip_groups": 							  {Tok: azureDataSource(azureNetwork, "getIpGroups")},
```

Comment:
Is this correct?

Reply:
https://github.com/hashicorp/terraform-provider-azurerm/pull/24540/files

Reply:
Looks correct to me.